File: learning/step01/step01.cpp
#include <iostream>

using namespace std;

int main () {
  int N;
  cout << "This is a simple program in C++\n";
  cout << "Enter a small integer: ";
  cin >> N;
  cout << "\nThe squares and cubes of the first " << N << " numbers are:\n";
  for (int i=1; i<N+1; i++) {
    cout << "Number: " << i << ". Square: " << i*i << ". Cube: " << i*i*i << endl;
  }
  return 0;
}

File: learning/step01/step01.py
print("This is a simple program in python")
N = int(raw_input("Enter a small integer: "))
print("The squares and cubes of the first "+str(N)+" numbers are:")
for i in range(1,N+1):
  print("Number: "+str(i)+". Square: "+str(i**2)+". Cube: "+str(i**3))

